test/Test/HTTPure/ResponseSpec.purs

```diff
@@ -2,15 +2,31 @@ module Test.HTTPure.ResponseSpec where
 
 import Prelude
 
+import Effect.Aff as Aff
 import Effect.Class as EffectClass
+import Node.Buffer as Buffer
+import Node.Encoding as Encoding
+import Partial.Unsafe as Unsafe
 import Test.Spec as Spec
 
+import HTTPure.Body as Body
 import HTTPure.Headers as Headers
 import HTTPure.Response as Response
 
 import Test.HTTPure.TestHelpers as TestHelpers
 import Test.HTTPure.TestHelpers ((?=))
 
+-- | Assert that a response body equals the given string
+bodyShouldEqual :: Body.Body -> String -> Aff.Aff Unit
```

Comment:
So I actually don't like using this helper, because the tests should be very precise, and using this helper requires anyone writing tests to remember to check the type of the body in addition to using it for the test to be accurate.  I'd prefer have a helper to check that a binary body equals a value, and use string equality directly for string bodies, and expect the individual tests to properly deconstruct and test their responses (using `case`).  Thoughts?